CHANGELOG.md

```diff
@@ -13,6 +13,18 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
 ### Fixed
 ### Security
 
+## [1.5.0] - 2017-10-25
+
+### Added
+- Adds an Annotation data model #765
+- Spotlight updates - search fields are configurable now to be turned "off" by default #796
+- Adds search_field for TOC, incipit, manuscript title, manuscript number, text_title #792
+### Changed
+- Updates mod indexing - incipit #782, displayLabel/title #785, collection format #793
+### Fixed
+- Removes IIIF badge from non-image content #794
```

Comment:
Not sure if this is a change or a fix but...
The IIIF badge was removed from non-image content in 1.4.0 in #745. #794 addresses the indexing performance issues caused by #745.

Reply:
Thanks for clarification on that. I've updated the PR!